lib/evss/error_middleware.rb

```diff
@@ -7,7 +7,7 @@ def on_complete(env)
       case env[:status]
       when 200
         resp = env.body
-        raise EVSSError, resp['messages'] if resp['success'] && !resp['success']
+        raise EVSSError, resp['messages'] if 'success'.in?(resp) && !resp['success']
```

Comment:
This could also be `resp['success'] == false`, but I think writing it this way expresses the intent more clearly.